src/knockout.contextmenu.js

```diff
@@ -1,15 +1,25 @@
-(function (undefined) {
-'use strict';
-
-if (typeof ko !== undefined + '' && typeof document !== undefined + '') {
-  bindContextMenu(ko, document);
-}
-
-if (typeof module !== undefined + '' && module.exports) {
-  module.exports = bindContextMenu;
+/* knockout.contextmenu v0.4.4
+   Nicolás Escalante - nlante@gmail.com
+   Issues: https://github.com/nescalante/knockout.contextmenu/issues
+   License: MIT */
+(function (factory) {
+  'use strict';
+
+  //if (define !== undefined && typeof define === 'function' && define.amd) {
+  //  // AMD. Register as an anonymous module.
+  //  define(['knockout'], factory);
```

Comment:
Because it breaks your tests.  Grunt and requirejs apparently don't play nicely together.  Do you have any idea how to run tests with this uncommented?